app/core/core/src/main/java/io/syndesis/core/Labels.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.core;
+
+import java.util.Locale;
+
+public final class Labels {
```

Comment:
I think we could unit test this quite easily. Any reason on not using/extending the existing class? https://github.com/syndesisio/syndesis/blob/814d0cdf3b443adf843f0fe1e0941331ae6a1068/app/core/core/src/main/java/io/syndesis/core/Names.java#L20